### PR TITLE
363 fe topic page

### PR DESCRIPTION
**Pull request by wayangalihpratama**
screencapture-localhost-3001-topics-2021-09-02-20_34_14

### CODE REVIEW

File: frontend/src/modules/topics/topic-images.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import imgCapacityBuilding from "../../images/topics/capacity-building.jpeg";
+import imgMarineLitter from "../../images/topics/marine-litter.jpeg";
+import imgPlastics from "../../images/topics/plastics.jpeg";
+import imgProductByDesign from "../../images/topics/product-by-design.jpeg";
+import imgSourceToSea from "../../images/topics/source-to-sea.jpeg";
+import imgWasteManagement from "../../images/topics/waste-management.jpeg";
+
+export const TopicImages = [
+  { id: 351, tag: "capacity building", image: imgCapacityBuilding },
```
Comment (punchagan) [inline frontend/src/modules/topics/topic-images.js:9]: Do we really need these IDs here?

File: frontend/src/modules/topics/topic.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            </span>
+            <Image
+              className="topic-item-image"
+              width="100%"
+              src={image || imageNotFound}
+              preview={false}
+            />
+            <div className="topic-item-title-wrapper">
+              <div className="topic-item-title">{tag}</div>
+              <Button className="topic-item-count" shape="circle" type="ghost">
```
Comment (punchagan) [inline frontend/src/modules/topics/topic.jsx:68]: I think we can get rid of the counts, for now. It is going to be tricky to get the counts before we - 1. make sure that Initiatives use the same tags as other resources (including IDs) and 2. we make sure that the topics names are not repeated in tags.  I'd say we get rid of the counts as part of the MVP.

File: frontend/src/modules/dropdown-menu/explore.jsx [hunk 3]
```diff
@@ -9,7 +9,11 @@ const ExploreDropdownMenu = withRouter(({ history, topics }) => {
       overlayClassName="menu-dropdown-wrapper"
       overlay={
         <Menu className="menu-dropdown">
-          <Menu.Item className="nav-link" disabled={!topics}>
+          <Menu.Item
+            className="nav-link"
+            disabled={!topics}
```
Comment (punchagan) [inline frontend/src/modules/dropdown-menu/explore.jsx:14]: Should we call `topics` as `topicsCount` instead?

**Review by punchagan — CHANGES_REQUESTED** [review]

File: frontend/src/modules/topics/topic-images.js (hunk 1 above)
Comment (wayangalihpratama) [reply frontend/src/modules/topics/topic-images.js:9]: Not really, 
I'll remove that

File: frontend/src/modules/topics/topic.jsx (hunk 2 above)
Comment (wayangalihpratama) [reply frontend/src/modules/topics/topic.jsx:68]: I'll comment the count code, for future development

**Comment by punchagan** [comment]
@wayangalihpratama I find that the history button doesn't work correctly. 

Here's what I tried. 

- Go to /topics
- Visit one of the topics page
- Click back to go back to /topics
- Click a different topic, but I appear to go back to the first topic I previously visited?

**Review by punchagan — APPROVED** [review]

**Pull request merged**